lgc/patch/PatchEntryPointMutate.cpp

```diff
@@ -1039,11 +1185,11 @@ void PatchEntryPointMutate::addUserDataArgs(SmallVectorImpl<UserDataArg> &userDa
       break;
 
     case ResourceNodeType::DescriptorTableVaPtr: {
-      // Check if the descriptor set is in use.
-      if (userDataUsage->descriptorSets.size() <= node.innerTable[0].set)
+      // Check if the descriptor set is in use. For compute with calls, enable it anyway.
+      if (!isComputeWithCalls() && userDataUsage->descriptorSets.size() <= node.innerTable[0].set)
         break;
       auto &descSetUsage = userDataUsage->descriptorSets[node.innerTable[0].set];
```

Comment:
This might access `userDataUsage->descriptorSets` out of bounds. (But this part will be rewritten anyway based on Nicolai’s comments.)